src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -749,12 +755,26 @@ def _plotGammaMarkerConstraint(self, x, y):
             x = min(x, vmax)
         return x, y
 
+    def setDataInPlotMode(self, mode: str | DataInPlotMode):
+        mode = DataInPlotMode.from_value(mode)
+        if mode is DataInPlotMode.HISTOGRAM:
+            action = self._dataHistogramAction
+        elif mode is DataInPlotMode.RANGE:
+            action = self._dataRangeAction
+        else:
+            raise NotImplementedError
```

Comment:
Missing instanciation + it's not a missing implementation:
```suggestion
            raise ValueError("Mode not supported")
```